docs/packaging/submitting-a-package.md

```diff
@@ -1,59 +1,90 @@
+Please refrain from submitting a pull request for the following instances:
 
-When contributing patches to Solus, please note that we do not accept the upload of binary files (i.e. `.eopkg`).
-All repository packages are built by a dedicated build server, from the source files provided in your patch upload.
+- For a package that is yet to be accepted for inclusion by a member of the Solus Staff team. Search [open package requests](https://github.com/getsolus/packages/issues?q=label%3A%22Package+Request%22) to see if there is an open request for the package.
+  We welcome you to politely reach out via the package request issue or our Support room on [Matrix](/docs/user/contributing/getting-involved#matrix-chat) if you deem the review of the request to be time-sensitive in nature.
+- If your pull request is a Work In Progress / WIP. Pull requests that are completed or are marked as *request for comments* (RFC) are accepted. For RFC request please ensure your patch title contains `[RFC]` and is marked as 'draft'. WIP patches clutter the issue tracker and make patch review by Solus Staff more time consuming and introduces unnecessary work.
 
-At minimum, your patch will include changes for the following:
+## Test Your Package
 
-- `package.yml`
-- `pspec_*.xml`
+After you ran `make` (or the appropriate command for the package) you now have `.eopkg` files in the package directory. Each pull request requires you to explain how you tested the package to make sure things work as expected. These files allow us to perform that testing.
 
-If any additional files were required for the build, then you must also include the `files/` directory.
-If you are creating a new package, you will also need to include a `Makefile` containing the following text:
+### Install the `.eopkg` Files
+To install your new package, run the following command in the directory where you built the package. Include **all** `.eopkg` files that were built.
 
+```bash
+sudo eopkg it *.eopkg
 ```
-include ../Makefile.common
+
+Testing need not be very complicated. The goal is to ensure the package will reliably work for our users. Testing effort will depend on how complex the software is. At minimum, you want to ensure the application launches, and performs basic functionality as expected.
+
+For example, if you were updating a text editor, you would want to make sure you can open, create and save files. If you know of issues the package has had in the past, it's a good idea to test that as well to make sure there are no regressions. There are plenty of historical pull requests to review if you want more detail.
+
+### Returning to the Repository Version (optional but recommended)
+
+It is a good idea to remove the package you built, and return to the repository version, once you are done testing. To do this, run the following command on the main application. Example:
+
+```bash
+sudo eopkg it --reinstall nano
 ```
```

Comment:
```suggestion
```bash
sudo eopkg it --reinstall tree
```
```

Reply:
Sure, since tree was mentioned in the doc about your first package update.